### PR TITLE
Linting:Add scss variable files to linting

### PR DESCRIPTION
**Pull request by geekman-rohit**
Adds jsass-vars scss variable files from scss/varibales/ folders to js linting.

### CODE REVIEW

File: scss/variables/chassis.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // This is just boiuler plate dont worry about it
 // This just supports loading in any enviroment
-(function ( root, factory ) {
+( function( root, factory ) {
     if ( typeof define === "function" && define.amd ) {
         define( [], factory );
```
Comment (geekman-rohit) [inline scss/variables/chassis.js]: Looks like this file has wrong indentation, skipped my notice, I'll fix that

**Comment by geekman-rohit** [comment]
@sfrisk , updated the PR

File: tasks/options/jscs.js [hunk 2]
```diff
@@ -1,3 +1,6 @@
 module.exports = {
-	all: [ "*.js", "performance/**/*.js" ]
-};
\ No newline at end of file
+	all: [ "*.js", "performance/**/*.js", "scss/variables/**/*.js" ],
+	options: {
+		fix: true
+	}
```
Comment (geekman-rohit) [inline tasks/options/jscs.js]: I ran it once with fix true no issues. We could get rid of the option now or do we want to keep it ? @arschmitz ?

File: scss/variables/template.js [hunk 3]
```diff
@@ -1,13 +1,14 @@
 // This is just boiuler plate dont worry about it
```
Comment (sfrisk) [inline scss/variables/template.js]: spelling error - "boiler plate" not "boiuler plate"

Comment (sfrisk) [inline scss/variables/template.js]: (This could easily be fixed in a different issue though)

File: tasks/options/jscs.js (hunk 2 above)
Comment (sfrisk) [inline tasks/options/jscs.js]: Per discussion with @arschmitz, fix should not be in the repo at all.

File: scss/variables/template.js (hunk 3 above)
Comment (geekman-rohit) [inline scss/variables/template.js]: its a small thing, ill do it in this PR.

File: tasks/options/jscs.js (hunk 2 above)
Comment (geekman-rohit) [inline tasks/options/jscs.js]: yep that shouldn't be here, I actually forgot to remove it. Thanks for reminding.

**Comment by geekman-rohit** [comment]
@sfrisk updated.

**Pull request closed without merging**